task2/src/pages/Home/components/MovieSearch/MovieSearch.jsx

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+import { AppButtonStyled } from '@components';
+
+const MovieSearchInputStyled = styled.input`
+    width: 80%;
+    height: 100%;
+    color: #ffffff;
+    background-color: rgba(0,0,0,0.7);
+    font-size: 16px;
+    border: none;
+    border-radius: 6px;
+    padding: 10px;
+`;
+
+const MovieSearchButtonStyled = styled(AppButtonStyled).attrs(props => ({
+    placeholder: props.searchPlaceholder,
```

Comment:
`searchPlaceholder` never passed here

I guess it's not necessary at all - lets talk

Reply:
Yep, just blind copy-paste